app/mailers/patient_mailer.rb

```diff
@@ -175,12 +183,14 @@ def assessment_email(patient)
     @patients = patient.active_dependents.uniq.collect do |dependent|
       { patient: dependent, jurisdiction_unique_id: Jurisdiction.find_by_id(dependent.jurisdiction_id).unique_identifier }
     end
+
+    update_last_assessment_reminder_sent(patient) # Update last send attempt timestamp before Twilio call
     mail(to: patient.email&.strip, subject: I18n.t('assessments.email.reminder.subject', locale: @lang || :en)) do |format|
       format.html { render layout: 'main_mailer' }
     end
     add_success_history(patient, patient)
-    # Always update the last contact time so the system does not try and send emails again.
-    patient.update(last_assessment_reminder_sent: DateTime.now)
+  rescue StandardError
+    clear_last_assessment_reminder_sent(patient) # Reset send attempt timestamp on failure
```

Comment:
This is probably out of scope of this PR but we should have an `add_failure_history_email` option in this case as well, since we have it for the all the others.

Reply:
That's a good idea, please make a ticket when you get a chance. We'll probably want to think about the language a bit since the causes of failure are different for email.

Reply:
Done: https://tracker.codev.mitre.org/browse/SARAALERT-1188